frontend/test/app/components/ExperimentTourDialog-test.js

```diff
@@ -26,13 +27,11 @@ describe('app/components/ExperimentTourDialog', () => {
       onCancel: sinon.spy()
     };
 
-    subject = shallow(<ExperimentTourDialog {...props} />);
+    subject = mount(<ExperimentTourDialog {...props} />);
   });
 
-  const findByL10nID = id => subject.findWhere(el => id === el.props()['data-l10n-id']);
-
-  it('should render expected default content', () => {
-    expect(JSON.parse(findByL10nID('tourOnboardingTitle').prop('data-l10n-args')).title)
+  it.skip('should render expected default content', () => {
```

Comment:
I think it was me who added these `skips` here when I was working on tests. Do you want to try to unskip them?